code/modules/antagonists/heretic/heretic_knowledge.dm

```diff
@@ -577,6 +585,8 @@
 	mutually_exclusive = TRUE
 	cost = 1
 	priority = MAX_KNOWLEDGE_PRIORITY - 10 // A pretty important midgame ritual.
+	depth = 6
+	research_tree_icon = icon('icons/obj/antags/eldritch.dmi',"book")
```

Comment:
And if we do end up caching them per type, then rather than making icons like this, we should just have `icon = 'icons/obj/antags/eldritch.dmi' icon_state = "book"`

Reply:
I imagine we also want a variable for frame and direction override, since sometimes you want a specific frame/direction of the icon to display (without it, the tgui displays a table of all the frames...)